policies/Demo_Course/policy.json

```diff
@@ -14,6 +14,7 @@
         "display_name": "Demonstration Course",
         "due": null,
         "end": null,
+        "enrollment_start": "2013-02-05T00:00:00Z",
```

Comment:
The only question I have why do you choose 2013? I would stay on 2022-02-24T04:00:00Z

Reply:
it is equal to the default course start [example from upstream](https://github.com/openedx/openedx-demo-course/blob/5a66489ec15b11f434567b2b39b3d74999bc4e69/policies/Demo_Course/policy.json#L475) and from [our fork](https://github.com/raccoongang/openedx-demo-course/blob/ef790d37bf69064b183af4bf65aa51c442d278b4/policies/Demo_Course/policy.json#L475)

I had the same question when opened the PR to the upstream (but about the course start) - I've got no answers on why 2013 and how/when it is updated 🙂

Reply:
we can use any date in the past but 2013 will be closer to the upstream version